$OpenBSD: patch-configure,v 1.13 2012/01/07 13:47:52 sthen Exp $

XXX careful! this must apply to pthread and not any other lib.

if updating, to include additional context to help avoid problems later,
use a hand-rolled patch or 'make update-patches DIFF_ARGS=-u6'.


--- configure.orig	Tue Nov  1 12:31:18 2011
+++ configure	Fri Jan  6 22:20:01 2012
@@ -10540,14 +10540,14 @@ return pthread_create ();
 }
 _ACEOF
 for ac_lib in '' pthread; do
   if test -z "$ac_lib"; then
     ac_res="none required"
   else
-    ac_res=-l$ac_lib
-    LIBS="-l$ac_lib  $ac_func_search_save_LIBS"
+    ac_res=-$ac_lib
+    LIBS="-$ac_lib  $ac_func_search_save_LIBS"
   fi
   if ac_fn_c_try_link "$LINENO"; then :
   ac_cv_search_pthread_create=$ac_res
 fi
 rm -f core conftest.err conftest.$ac_objext \
     conftest$ac_exeext
